synapse/handlers/auth.py

```diff
@@ -627,23 +627,28 @@ async def check_ui_auth(
 
     async def add_oob_auth(
         self, stagetype: str, authdict: Dict[str, Any], clientip: str
-    ) -> bool:
+    ) -> Any:
```

Comment:
It looks like the result of this isn't even used anymore after this?

If it is, I'd like to see a more specific type here than `Any`. If it isn't, we should have it return `None` and properly document what it raises.

Reply:
Yep, I'll do that

Reply:
How's that?